opm/simulators/flow/FlowMain.hpp

```diff
@@ -32,6 +32,9 @@
 #include <opm/simulators/flow/FlowUtils.hpp>
 #include <opm/simulators/flow/SimulatorFullyImplicitBlackoil.hpp>
 
+#if HAVE_MPI
+#define RESERVOIR_COUPLING_ENABLED
+#endif
```

Comment:
Was this needed to make serial compilation work? Could we hide this somewhere e.g. by providing methods doing nothing in this case are throwing? Might make the code clearer and cleaner.

Reply:
Yes this was done to make serial compilation work. I think it is clearer than using `#if HAVE_MPI`. Instead we use `#ifdef RESERVOIR_COUPLING_ENABLED` which make it clearer. I am not sure I understood your comment about hiding it? Did you mean moving it to a designated header file?